### DISCUSSION TITLE
go install goreleaser: package cmp is not in GOROOT

### DISCUSSION THREAD

**Discussion by iTanken** [goreleaser/goreleaser#4443] [Q&A] [answered]
### go version

go version go1.20.11 windows/amd64

<details>
<summary>go env</summary>

set GO111MODULE=auto
set GOARCH=amd64
set GOBIN=
set GOCACHE=E:\Workspace\Go\go-build
set GOENV=E:\Workspace\Go\env
set GOEXE=.exe
set GOEXPERIMENT=
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOINSECURE=
set GOMODCACHE=E:\Workspace\Go\pkg\mod
set GOOS=windows
set GOPATH=E:\Workspace\Go
set GOPROXY=https://goproxy.io,direct
set GOROOT=D:\Dev\Go\SDK\go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=D:\Dev\Go\SDK\go\pkg\tool\windows_amd64
set GOVCS=
set GOVERSION=go1.20.11
set GCCGO=gccgo
set GOAMD64=v1
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=
set GOWORK=
set CGO_CFLAGS=-O2 -g
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-O2 -g
set CGO_FFLAGS=-O2 -g
set CGO_LDFLAGS=-O2 -g
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=C:\Users\User\AppData\Local\Temp\go-build4244915004=/tmp/go-build -gno-record-gcc-switches

</details>

### go install

```shell
> go install github.com/goreleaser/goreleaser@latest
E:\Workspace\Go\pkg\mod\github.com\charmbracelet\x\exp\ordered@v0.0.0-20231010190216-1cb11efc897d\ordered.go:3:8: package cmp is not in GOROOT (D:\Dev\Go\SDK\go\src\cmp)
```

**Comment by caarlos0** [comment, marked as answer]
goreleaser needs go 1.21

you can use any other of the install options though: https://goreleaser.com/install/

**Reply by iTanken** [reply]
Okay, thank you.

So

- ...
- go 1.17 : goreleaser@v1.6.3
- go 1.18 : goreleaser@v1.10.3
- go 1.19 : goreleaser@v1.15.1
- go 1.20 : goreleaser@v1.21.2
- go 1.21 : goreleaser@v1.22.1
- ...

Is this the case?

**Reply by caarlos0** [reply]
I don't remember exactly, but usually each release uses the last Go version available at compile time.

I should probably do a better job documenting that though, will add it to the /install page too
